# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/MacroAssembler.h
# Commit: 315a592cfd3c
# Full Hash: 315a592cfd3ca405854ea9b790e432a28e9b8836
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2020-10-31 21:05:17
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Part 7: Remove Frame::tls. r=lth
#   
#   We remove Frame::tls and fix prologue/epilogue. Runtime, Ion and Baseline are ready for this move already.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D83064
# ==============================================================================

diff -r 1f21e8f610a7 -r 315a592cfd3c js/src/jit/MacroAssembler.h
--- a/js/src/jit/MacroAssembler.h	Sat Oct 31 09:56:05 2020 +0000
+++ b/js/src/jit/MacroAssembler.h	Sat Oct 31 09:56:06 2020 +0000
@@ -3556,8 +3556,6 @@
   void guardStringToInt32(Register str, Register output, Register scratch,
                           LiveRegisterSet volatileRegs, Label* fail);
 
-  void loadWasmTlsRegFromFrame(Register dest = WasmTlsReg);
-
   template <typename T>
   void loadTypedOrValue(const T& src, TypedOrValueRegister dest) {
     if (dest.hasValue()) {